tools/bricks/feature/hooks/post_gen.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:io';
+
+import 'package:mason/mason.dart';
+
+Future<void> run(HookContext context) async {
+  final process = await Process.start(
+    'fvm',
+    ['dart', 'format', '-l', '100', 'lib', 'test'],
```

Comment:
There are many places where this is configured. it's better to make a script `scripts/format.sh` and call it in needed places